README.md

```diff
@@ -1,2 +1,50 @@
-# setup-testspace
-A GitHub Action to install and configure the Testspace client used for publishing test content
+
+![Testspace Action](https://github.com/testspacehq/setup-testspace/workflows/Testspace%20Action/badge.svg)
+
+# Testspace client Setup JavaScript Action
+A GitHub Action used to install and configure the Testspace client used for publishing test content to [Testspace.com](https://testspace.com). 
+
+## Usage
+Setting up the Testspace client:
+
+```yaml
+uses: testspace-com/setup-testspace@v1
```

Comment:
I personally don't like `testspace-com`. I see `-com` as a band-aid to name.

Reply:
Well I understand, but there is no plan to use s2. The other alternative was `testspacehq`, which you did not like either.  

Few examples:
 -  https://github.com/marketplace/restyled-io (https://github.com/restyled-io)
 - https://github.com/marketplace/gitpod-io (https://github.com/gitpod-io)

Note, we also own `testspace.io`.

Reply:
> Well I understand, but there is no plan to use s2.

I guess you already have made a decision on an exact plan? What is that plan? 
As said elsewhere, since we have two components - App and Action - they both should be sourced from the same location.

> BTW I am going to try to contact `testspace` using an public Issue trick (`@testspace would you give up ..`)

That is very good idea.

Reply:
I am going to assume I will **not** hear anything back from "@testspace", but if we do will use it. 

I want to go with `testspace-io`. I agree we should then use this org to list the Application (i.e. `Testspace.io`).